src/components/Card.js

```diff
@@ -29,12 +41,26 @@ export default class Card {
   }
 
   _handleLikeIcon() {
-    this._likeButton.classList.toggle("card__like-button_active");
+    // this._likeButton.classList.toggle("card__like-button_active");
+    if (this._isLiked) {
+      this._api.dislikeCard(this._id).then(() => {
+        this._isLiked = false;
+        this._likeButton.classList.remove("card__like-button_active");
+      });
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**